charts/vela-core/templates/defwithtemplate/notification.yaml

```diff
@@ -17,6 +17,38 @@ spec:
         )
 
         parameter: {
+        	lark?: {
+        		// +usage=Specify the the lark url, you can either sepcify it in value or use secretRef
+        		url: {
+        			value: string
+        		} | {
+        			secretRef: {
+        				// +usage=name is the name of the secret
+        				name: string
+        				// +usage=key is the key in the secret
+        				key: string
+        			}
+        		}
+        		// +useage=Specify the message that you want to sent
+        		message: {
+        			// +usage=msgType can be text, post, image, interactive, share_chat, share_user, audio, media, file, sticker
+        			msg_type: string
```

Comment:
msgType vs msg_type

Reply:
Thx for review, according to lark api, the key name should be msg_type, i will fix the annotation from msgType to msg_type